webapp/src/js/components/containers/DataTableWithActions.js

```diff
@@ -184,16 +199,99 @@ let DataTableWithActions = React.createClass({
     }
   },
 
+  handleSearchOpen() {
+    this.setState({searchOpen: true});
+  },
+
+  handleSearchChange(event) {
+
+    let searchText = event.target.value;
+
+    this.setState({search: searchText});
+
+    if (searchText === '') {
```

Comment:
I'm finding it counter-intuitive that I can have either a filter or a text search. An example would be in AG1000G where the default query is for only high quality variants. If I then entered a text search it would show me results from all variants. I strongly feel that search should be a prop, translate that prop to a search_query and then pass "query AND search_query" down to the child DataTableView. Does that make sense?! Otherwise I feel there is a hidden interaction between query and search.